src/local/index.coffee

```diff
@@ -12,9 +12,10 @@ Watcher = require './watcher'
 # It also applied changes from the remote cozy on the local filesystem.
 class Local
     constructor: (config, @prep, @pouch) ->
+        deviceName = config.getDefaultDeviceName()
         @basePath = config.getDevice().path
         @tmpPath  = path.join @basePath, ".cozy-desktop"
-        @watcher  = new Watcher @basePath, @prep, @pouch
+        @watcher  = new Watcher @basePath, @prep, @pouch, deviceName
```

Comment:
it's for that I don't add @ before deviceName.
You prefer this: 

    @watcher  = new Watcher @basePath, @prep, @pouch, config.getDefaultDeviceName()